src/coreclr/jit/importer.cpp

```diff
@@ -10593,6 +10593,12 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 JITDUMP(" %08X", resolvedToken.token);
 
                 lclTyp = JITtype2varType(info.compCompHnd->asCorInfoType(resolvedToken.hClass));
+
+                if (lclTyp == TYP_STRUCT)
+                {
+                    lclTyp = impNormStructType(resolvedToken.hClass);
```

Comment:
TP nit: if we have a struct, we can get its layout here, check `layout->GetType()` for SIMD, and then use it below. This will save on the redundant `impNormStructType` call. `impNormStructType` is not incredibly cheap.